components/timeline/spec/Timeline.stories.mdx

```diff
@@ -74,12 +74,12 @@ A standard Timeline.
   </Story>
 </Preview>
 
-### NoProps
+### No Props
 
-A standard Timeline.
+A timeline component without an item prop will not render
 
 <Preview>
   <Story name="NoProps">
-    <Timeline/>
+    <Timeline />
   </Story>
 </Preview>
```

Comment:
I'm not sure this has any value as a story: https://6088298134131d650ffca317--hods-poc.netlify.app/components?name=Timeline#NoProps-example-react

I'd just delete it. We don't need to document unhappy paths.

In fact I'd suggest we just make the `items` prop mandatory. (If it isn't already.)